test/babel-eslint.js

```diff
@@ -71,7 +71,7 @@ function parseAndAssertSame(code) {
   // assert.equal(esAST, babylonAST);
 }
 
-describe("babylon-to-esprima", () => {
+describe("babylon-to-espree", () => {
```

Comment:
I think this is correct? I'm assuming this is a holdover from a long time ago :)